connect/cli/plugins/project/report/helpers.py

```diff
@@ -116,105 +122,128 @@ def add_report(project_dir, package_name):
-    )
-
-    template_name = os.path.splitext(os.path.basename(template))[0]
-
-    repo_dir = os.path.join(config_dict['cookiecutters_dir'], template_name)
-    if os.path.isdir(repo_dir):
-        purge_dir(repo_dir)
-
-    repo_dir, _ = determine_repo_dir(
-        template=template,
-        abbreviations=config_dict['abbreviations'],
-        clone_to_dir=config_dict['cookiecutters_dir'],
-        checkout=None,
-        no_input=False,
-        password=None,
-        directory=None,
-    )
-
-    context_file = os.path.join(repo_dir, 'cookiecutter.json')
-    context = generate_context(
-        context_file=context_file,
-        default_context=config_dict['default_context'],
-        extra_context=None,
-    )
-
     answers = dialogus(
         ADD_REPORT_QUESTIONS,
-        'Add report to existing project',
+        'Reports add project',
         intro=REPORT_ADD_WIZARD_INTRO,
         summary=REPORT_SUMMARY,
-        finish_text='Add',
+        finish_text='Create',
         previous_text='Back',
     )
 
     if not answers:
         raise ClickException('Aborted by user input')
 
-    context['cookiecutter']['_template'] = template
-    context['cookiecutter']['project_slug'] = project_slug
-    context['cookiecutter']['package_slug'] = package_slug
-    context['cookiecutter']['initial_report_name'] = answers['initial_report_name']
-    context['cookiecutter']['initial_report_slug'] = answers['initial_report_slug']
-    context['cookiecutter']['initial_report_description'] = answers['initial_report_description']
-    context['cookiecutter']['initial_report_renderer'] = answers['initial_report_renderer']
-
-    result = _generate_files(context, output_dir, repo_dir)
-
-    return result, context['cookiecutter']['initial_report_slug']
+    report_path = os.path.join(project_path, answers['initial_report_slug'])
+    if os.path.exists(report_path):
+        raise ClickException(
+            f'\nThe report directory called `{report_path}` already exists, '
+            '\nplease, choose other report name or delete the existing one.',
+        )
 
+    template_folder = os.path.join(os.path.dirname(__file__), 'templates', 'add')
+    renderer = BoilerplateRenderer(
+        context=answers,
+        template_folder=template_folder,
+        output_dir=project_path,
+        post_render=functools.partial(
+            generate_empty_xlsx,
+            answers['initial_report_slug'],
+        ),
+    )
+    renderer.render()
 
-def _generate_files(context, output_dir, repo_dir):
-    result = generate_files(
-        repo_dir=repo_dir,
-        context=context,
-        overwrite_if_exists=False,
-        skip_if_file_exists=False,
-        output_dir=output_dir,
+    _add_report_to_descriptor(
+        project_dir=project_dir,
+        package_dir=package_name,
+        context=renderer.context,
     )
-    return result
+
+    console.secho('\nReport has been successfully created.', fg='blue')
```

Comment:
Please change the color from blue to green.
Also the message can be changed to:

`Report {report_name} has been successfully added to the project {project_name}. `